addons/sys_external/fnc_stopUsingExternalRadio.sqf

```diff
@@ -0,0 +1,42 @@
+/*
+ * Author: ACRE2Team
+ * Stop using an external radio, either returning it to the owner or giving it to another player. If target
+ * is skipped, a default action to return to the owner is taken
+ *
+ * Arguments:
+ * 0: Unique radio identity <STRING>
+ * 1: New end user/original owner <OBJECT>(Optional)
```

Comment:
`... <OBJECT> (default: objNull)`
It's not actually optional per `params` though.